app/ai-gateway/monitor-ai-llm-metrics.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Monitor AI LLM metrics"
+content_type: reference
+layout: reference
+
+products:
+  - gateway
+
+tags:
+  - ai-gateway
+  - monitoring
+
+plugins:
+  - prometheus
+  - ai-proxy
+  - ai-proxy-advanced
+
+min_version:
+  gateway: '3.7'
+
+description: "This guide walks you through collecting AI metrics and sending them to Prometheus."
+
+related_resources:
+  - text: Kong AI Gateway
+    url: /ai-gateway/
+  - text: Kong AI Gateway plugins
+    url: /plugins/?category=ai
+  - text: Status API
+    url: /api/gateway/status/v1/
+  - text: Admin API
+    url: /api/gateway/admin-ee/3.9/
```

Comment:
```suggestion
  - text: Admin API
    url: /api/gateway/admin-ee/
  - text: Prometheus plugin
    url: /plugins/prometheus/
```

Reply:
I didn't add Prometheus because it's already in the list of plugins